src/sentry/web/urls.py

```diff
@@ -425,11 +424,6 @@ def init_all_applications():
         ProjectIssueTrackingView.as_view(),
         name='sentry-project-issue-tracking'
     ),
-    url(
-        r'^(?P<organization_slug>[\w_-]+)/(?P<project_slug>[\w_-]+)/settings/plugins/$',
-        ProjectPluginsView.as_view(),
```

Comment:
Another way we can do this is to use `react_page_view`, which allows us to keep django's named routes, but not sure if this is desired. @macqueen thoughts?